lib/atrium/member.rb

```diff
@@ -134,7 +134,15 @@ def update(params)
 
     def credentials
       endpoint = "/users/#{user_guid}/members/#{guid}/credentials"
-      ::Atrium.client.make_request(:get, endpoint)
+      credential_response = ::Atrium.client.make_request(:get, endpoint)
+
+      return nil if credential_response.nil?
```

Comment:
Should this be `return [] unless credential_response.present?`

Reply:
Not sure. I based it on the  `def challenges` in member.rb

I can see why we would want to return [] over nil. Do you think we should we do that in `def challenges` as well?

Reply:
Ah gotcha. In that case, I think we're fine as long as we're consistent.